advanced_functionality/multi_model_bring_your_own/multi_model_endpoint_bring_your_own.ipynb

```diff
@@ -11,7 +11,7 @@
     "\n",
     "For the inference container to serve multiple models in a multi-model endpoint, it must implement [additional APIs](https://docs.aws.amazon.com/sagemaker/latest/dg/build-multi-model-build-container.html) in order to load, list, get, unload and invoke specific models. This notebook demonstrates how to build your own inference container that implements these APIs.\n",
     "\n",
-    "**Note**: Because this notebook builds a Docker container, it does not run in Amazon SageMaker Studio.\n",
+    "**Note**: the section of code that build the container, has been changed to allow enable build of a Docker container in Amazon SageMaker Studio, using 'sagemaker-studio-image-build'.\n",
```

Comment:
**Note**: To build a Docker container in Amazon SageMaker Studio, we use the 'sagemaker-studio-image-build' package and its `sm-docker build` command.